Makefile

```diff
@@ -1,4 +1,4 @@
-CC ?= gcc
+CC ?= clang++
```

Comment:
can we keep the default to `gcc` I think that's more common

Reply:
done